compliance/notify.py

```diff
@@ -892,6 +892,16 @@ def notify(self):
         for accreditation, desc in messages:
             if accreditation not in self._config or not desc:
                 continue
+            # determine if we should page for all failing checks in this
```

Comment:
This new config should get documented in https://github.com/ComplianceAsCode/auditree-framework/blob/main/doc-source/notifiers.rst. Note, however, that the pagerduty notifier is not yet documented there. I think you can just do a brief description and give an example of the two ways to configure (i.e. using a string, and using a dictionary).